core/src/main/scala/cromwell/util/StopAndLogSupervisor.scala

```diff
@@ -10,6 +10,7 @@ trait StopAndLogSupervisor { this: Actor =>
   final val stopAndLogStrategy: SupervisorStrategy = {
     def stoppingDecider: Decider = {
       case e: Exception =>
+        e.printStackTrace()
```

Comment:
I'm a little confused by this whole class. Despite the name it doesn't seem to have been doing any degree of logging previously. And now there's the `printStackTrace`, but does that actually get captured by logback, or only to the console?

Reply:
Oh right yeah this is me debugging. I'm going to try to find a way to log it properly though because when the worker actor crashes it doesn't say why which is annoying